scripts/amd/gemm/tune_gemm.py

```diff
@@ -348,30 +389,33 @@ def main():
         f_kernel[fi].close()
 
 
-def extract_kernel_time(M, N, K, config, df):
-    configStr, _ = gen_kernel_and_configStr_from_config(M, N, K, config, None, None, None)
+def extract_kernel_time(M, N, K, config, df, bias_size):
+    configStr, _ = gen_kernel_and_configStr_from_config(M, N, K, config, None, None, None, bias_size)
     df = df[df['KernelName'].str.contains(configStr)]
     meanTime = df['DurationNs'].tail(100).mean()
     return config, meanTime
 
 
-def profile_batch_kernels(M, N, K, gpuid, gpus, jobs, verbose):
+def profile_batch_kernels(M, N, K, gpuid, gpus, jobs, verbose, rotating_buffer_size):
```

Comment:
If we set the default value of `rotating_buffer_size` correctly in the generated_kernelM-N-K-0.py, we won't need `rotating_buffer_size` in this function at all.

Reply:
right, removed this parameter.